requirements.txt

```diff
@@ -7,5 +7,5 @@ nlohmann/json@v3.11.2 -DJSON_MultipleHeaders=ON -DJSON_BuildTests=Off
 ROCm/FunctionalPlus@v0.2.18-p0
 ROCm/eigen@3.4.0
 ROCm/frugally-deep@9683d557eb672ee2304f80f6682c51242d748a50
-ROCm/composable_kernel@15baccf2ecad4fb3498b8acb6bbf58fb5359c7a5 -DCMAKE_BUILD_TYPE=Release -DINSTANCES_ONLY=ON
+ROCm/composable_kernel@1dd9875c9ac783cabc8a536e0802b58d43b6b107 -DCMAKE_BUILD_TYPE=Release -DINSTANCES_ONLY=ON
```

Comment:
Can this be aligned with https://github.com/ROCm/MIOpen/pull/3142 ?

Reply:
Yes, I also recommend picking CK commit hashes from its `amd-develop` branch.